recipes/gembs/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "conda_v2" %}
```

Comment:
it would be best if the version of the package reflected the actual version of the software.

Reply:
Thank you for pointing this out. I'll have to ask for an up to date release. I'll definitely change this.